### ISSUE TITLE
The new-cap rule should allow the use of Date.UTC without `new`

### ISSUE THREAD

**Issue by dbrockman** [eslint/eslint#1211]
The `Date.UTC` function is not a constructors even though is begins with a capital letter. The `new-cap` rule will flag the use of `Date.UTC` when not preceded by the `new` keyword.

eslint version: 0.7.4

Example code that will be flagged by `new-cap` but should pass without a warning:

``` js
var timestamp = Date.UTC(2004, 02, 14);
```

I have submitted a pull request that fixes this issue: #1201
